### ISSUE TITLE
Obtain the probability p of the reward function saying a response A > response B

### ISSUE THREAD

**Issue by vermashresth** [vermashresth/RLAIF-KL-Error#2]
Also explore multiple options of how to get this probability - through Chain of Thought, or other tricks to obtain logits of tokens

**Comment by mauriciogtec** [comment]
Solved in #11
